import traceback
import numpy as np


def main():
    """
    Simply generates a random numpy array and prints it to stdout.
    """
    a = np.random.rand(3,2)
    print(a)


def sys_main():
    """
    Runs the main function using the system cli arguments, and
    returns a system error code.

    :return: 0 for success, 1 for failure.
    :rtype: int
    """

    try:
        main()
        return 0
    except Exception:
        print(traceback.format_exc())
        return 1


if __name__ == "__main__":
    try:
        main()
    except Exception:
        print(traceback.format_exc())

